docs/configuration/SUMMA_model_decisions.md

```diff
@@ -3,3 +3,396 @@
+
+| Option | Description |
+|---|---|
+| presTemp | **TODO: Describe presTemp <br> [Reference](http://doi.org/)** |
+| nrg_flux | **TODO: Describe nrg_flux <br> [Reference](http://doi.org/)** |
+| zeroFlux | **TODO: Describe zeroFlux <br> [Reference](http://doi.org/)** |
+
+<a id="bcLowrTdyn"></a>
+## 22. bcLowrTdyn
+Lower boundary condition for thermodynamics
+
+| Option | Description |
+|---|---|
+| presTemp | **TODO: Describe presTemp <br> [Reference](http://doi.org/)** |
+| zeroFlux | **TODO: Describe zeroFlux <br> [Reference](http://doi.org/)** |
+
+
+<a id="bcUpprSoiH"></a>
+## 23. bcUpprSoiH
+Upper boundary condition for soil hydrology
+
+| Option | Description |
+|---|---|
+| presHead | **TODO: Describe presHead <br> [Reference](http://doi.org/)** |
+| liq_flux | **TODO: Describe liq_flux <br> [Reference](http://doi.org/)** |
+
+
+<a id="bcLowrSoiH"></a>
+## 24. bcLowrSoiH
+Lower boundary condition for soil hydrology
+
+| Option | Description |
+|---|---|
+| presHead | **TODO: Describe presHead <br> [Reference](http://doi.org/)** |
+| bottmPsi | **TODO: Describe bottmPsi <br> [Reference](http://doi.org/)** |
+| drainage | **TODO: Describe drainage <br> [Reference](http://doi.org/)** |
+| zeroFlux | **TODO: Describe zeroFlux <br> [Reference](http://doi.org/)** |
+
+<a id="veg_traits"></a>
+## 25. veg_traits
+Parameterization for vegetation roughness length and displacement height
+
+| Option | Description |
+|---|---|
+| Raupach_BLM1994 | **TODO: Describe Raupach_BLM1994 <br> [Raupach, 1994](http://doi.org/10.1007/BF00709229)** |
+| CM_QJRMS1988 | **TODO: Describe CM_QJRMS1988 <br> [Choudhury and Monteith, 1988](http://doi.org/10.1002/qj.49711448006)** |
+| vegTypeTable | **TODO: Describe vegTypeTable <br> [Reference](http://doi.org/)** |
+
+<a id="rootProfil"></a>
+## 26. rootProfil
+Parameterization for the rooting profile
+
+| Option | Description |
+|---|---|
+| powerLaw | **TODO: Describe powerLaw <br> [Reference](http://doi.org/)** |
+| doubleExp | **TODO: Describe doubleExp <br> [Reference](http://doi.org/)** |
+
+
+<a id="canopyEmis"></a>
+## 27. canopyEmis
+Parameterization for canopy emissivity
+
+| Option | Description |
+|---|---|
+| simplExp | **TODO: Describe simplExp <br> [Reference](http://doi.org/)** |
+| difTrans | **TODO: Describe difTrans <br> [Reference](http://doi.org/)** |
+
+
+<a id="snowIncept"></a>
+## 28. snowIncept
+Parameterization for snow interception
+
+| Option | Description |
+|---|---|
+| stickySnow | **TODO: Describe stickySnow <br> [Reference](http://doi.org/)** |
+| lightSnow | **TODO: Describe lightSnow <br> [Reference](http://doi.org/)** |
+
+
+<a id="windPrfile"></a>
+## 29. windPrfile
+Canopy wind profile
+
+| Option | Description |
+|---|---|
+| exponential | **TODO: Describe exponential <br> [Reference](http://doi.org/)** |
+| logBelowCanopy | **TODO: Describe logBelowCanopy <br> [Reference](http://doi.org/)** |
+
+
+<a id="astability"></a>
+## 30. astability
+Stability function
```

Comment:
Should be explicit in what the stability is calculated for

Reply:
Same as above